drivers/pinmux/stm32/pinmux_board_nucleo_l476rg.c

```diff
@@ -36,16 +36,26 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PA0, STM32L4X_PINMUX_FUNC_PA0_PWM2_CH1},
 #endif /* CONFIG_PWM_STM32_2 */
 #ifdef CONFIG_SPI_1
+	/* SPI1 on the Arduino connectors pins A2, D3, D12, D11 */
 	{STM32_PIN_PA4, STM32L4X_PINMUX_FUNC_PA4_SPI1_NSS},
+	/* SPI1_SCK should output on PA5, but is used for LD2 */
 	{STM32_PIN_PB3, STM32L4X_PINMUX_FUNC_PB3_SPI1_SCK},
-	{STM32_PIN_PA5, STM32L4X_PINMUX_FUNC_PA5_SPI1_SCK},
 	{STM32_PIN_PA6, STM32L4X_PINMUX_FUNC_PA6_SPI1_MISO},
 	{STM32_PIN_PA7, STM32L4X_PINMUX_FUNC_PA7_SPI1_MOSI},
 #endif /* CONFIG_SPI_1 */
+#ifdef CONFIG_SPI_2
+	/* SPI2 on the ST Morpho Connector CN10 pins 16, 30, 28, 26*/
+	{STM32_PIN_PB12, STM32L4X_PINMUX_FUNC_PB12_SPI2_NSS},
+	{STM32_PIN_PB13, STM32L4X_PINMUX_FUNC_PB13_SPI2_SCK},
+	{STM32_PIN_PB14, STM32L4X_PINMUX_FUNC_PB14_SPI2_MISO},
+	{STM32_PIN_PB15, STM32L4X_PINMUX_FUNC_PB15_SPI2_MOSI},
+#endif /* CONFIG_SPI_1 */
 #ifdef CONFIG_SPI_3
-	{STM32_PIN_PB3, STM32L4X_PINMUX_FUNC_PB3_SPI3_SCK},
-	{STM32_PIN_PB4, STM32L4X_PINMUX_FUNC_PB4_SPI3_MISO},
-	{STM32_PIN_PB5, STM32L4X_PINMUX_FUNC_PB5_SPI3_MOSI},
+	/* SPI3 on the ST Morpho Connector CN7 pins 17, 1, 2, 3*/
+	{STM32_PIN_PA15, STM32L4X_PINMUX_FUNC_PA15_SPI3_NSS},
```

Comment:
@MaureenHelm, @superna9999: This is commonly used and I haven't seen messages where it could be not recommended (within ST)